src/test/java/org/broadinstitute/hellbender/tools/reference/CheckReferenceCompatibilityIntegrationTest.java

```diff
@@ -209,4 +212,48 @@ public void testReferenceCompatibilityNoBAMOrVCF() throws IOException {
         final String[] args = new String[]{"-refcomp", ref1.getAbsolutePath()};
         runCommandLine(args);
     }
+
+    @DataProvider(name = "cloudInputData")
+    public Object[][] cloudInputData() {
+        String testBucket = getGCPTestInputPath() + "org/broadinstitute/hellbender/tools/reference/";
+        return new Object[][]{
+                // dict, list of refs, expected output, isBAM, isVCF
+                new Object[]{testBucket + "reads_data_source_test1_withmd5s.bam",
+                        Arrays.asList(testBucket + "hg19mini.fasta"),
+                        new File(getToolTestDataDir(), "expected.testReferenceCompatibilityBAMWithMD5s_exactmatch.table"),
+                        true, false
+                },
+                new Object[]{ testBucket + "example_variants_withSequenceDict_withmd5.vcf",
+                        Arrays.asList(testBucket + "hg19mini.fasta"),
+                        new File(getToolTestDataDir(), "expected.testReferenceCompatibilityVCFWithMD5s.table"),
+                        false, true
+                },
+        };
+    }
+
+    @Test(dataProvider = "cloudInputData"/*, groups = "bucket"*/)
```

Comment:
@orlicohen `groups = "bucket"` Needs to be uncommented here and in the other test before these will be able to pass on github

Reply:
fixed